pkgs/top-level/python-packages.nix

```diff
@@ -26423,4 +26423,75 @@ in modules // {
     };
   };
 
+  sigtools = buildPythonPackage rec {
+    name = "sigtools-${version}";
+    version = "1.1a3";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/sigtools/${name}.tar.gz";
+      sha256 = "190w14vzbiyvxcl9jmyyimpahar5b0bq69v9iv7chi852yi71w6w";
+    };
+
+    # Tests fail with : AttributeError: 'module' object has no attribute 'test_forwards'
+    doCheck = false;
```

Comment:
Ok, so the problem was that it was using another tool for testing and downloading test dependencies (tox), which wasn't mentionned in the pypi archive (the configuration file for tests isn't included in it).
So there are two possibilities (other than disabling the tests)

- Get the archive from de github repo, which contains the `tox.ini` file used by tox and use it to run the tests.
- Use the archive from pypi, and explicitly run the tests listed in the tox files

I'd prefer the first option as it is much easier to maintain, but it seems to me that the policy is to avoid using github links when packages are on pypi, so before updating the pr, I'd like to have the opinion of someone more used to package python expressions for nix.